substrate/frame/support/src/dispatch.rs

```diff
@@ -700,6 +723,23 @@ mod weight_tests {
 			type DbWeight: Get<crate::weights::RuntimeDbWeight>;
 		}
 
+		#[pallet::genesis_config]
+		pub struct GenesisConfig<T: Config> {
+			#[serde(skip)]
+			pub _config: core::marker::PhantomData<T>,
+		}
+
+		impl<T: Config> Default for GenesisConfig<T> {
+			fn default() -> Self {
+				Self { _config: Default::default() }
+			}
+		}
+
+		#[pallet::genesis_build]
+		impl<T: Config> BuildGenesisConfig for GenesisConfig<T> {
+			fn build(&self) {}
```

Comment:
What do you need the empty genesis config for?

Reply:
It was a copy-paste thing. I removed it.